kong/plugins/rate-limiting/schema.lua

```diff
@@ -53,6 +53,9 @@ return {
           { redis_host = typedefs.host },
           { redis_port = typedefs.port({ default = 6379 }), },
           { redis_password = { type = "string", len_min = 0 }, },
+          { redis_ssl = { type = "boolean", default = false }, },
+          { redis_ssl_verify = { type = "boolean", default = false }, },
+          { redis_server_name = { type = "string" }, },
```

Comment:
```suggestion
          { redis_server_name = { typedefs.sni },
```

Reply:
(this uses one of our ready-to-use schema definitions, check this out: https://github.com/Kong/kong/blob/master/kong/db/schema/typedefs.lua#L515-L524)